types/sodium-native/sodium-native-tests.ts

```diff
@@ -135,4 +135,4 @@ sodium.crypto_box_keypair(publicKey, secretKey);
 const message = sodium.sodium_malloc(4);
 const cipherText = sodium.sodium_malloc(message.length + sodium.crypto_box_SEALBYTES);
 sodium.crypto_box_seal(cipherText, message, publicKey);
-sodium.crypto_box_seal_open(message, cipherText, publicKey, secretKey);
+const success = sodium.crypto_box_seal_open(message, cipherText, publicKey, secretKey);
```

Comment:
You can also add a type assertion here to make sure that this change doesn't regress:

```suggestion
// $ExpectType boolean
const success = sodium.crypto_box_seal_open(message, cipherText, publicKey, secretKey);
```

Reply:
Thanks very much for the tip. I didn't even know such a thing was available. I'm obviously new to this. Will do so tomorrow.